std/traits.d

```diff
@@ -3279,19 +3280,7 @@ alias Identity(alias A) = A;
    Yields $(D true) if and only if $(D T) is an aggregate that defines
    a symbol called $(D name).
  */
-template hasMember(T, string name)
```

Comment:
From this line I understand that it's by design. IRC @andralex suggested to add a separate introducesMember for cases where a particular member is not in the list of allMembers but introduced otherwse e.g. via inheritance or opDispatch.